mappings/servers/hvbc/metadata.json

```diff
@@ -0,0 +1,30 @@
+{
+    "id": "hvbc",
+    "name": "HVBC",
+    "description": "A practice server for legitimate players",
```

Comment:
You went over the 40-char limit

You should replace `legitimate` with `legit` probably